tests/src/Kernel/fixtures/markup_rendering_patterns/banner.yml

```diff
@@ -360,14 +467,49 @@ image_shade_page_banner_centered_with_link_full_width:
   assertions:
     count:
       'div[data-drupal-selector=banner]': 1
-      'div.bcl-banner': 1
+      'div.bcl-banner.bg-lighter': 1
+      'div.full-width': 1
+      'div.text-center': 1
+      'a': 1
+      'div.hero': 0
+      'div.overlay': 0
+      'div.shade': 1
+      'div.bcl-banner__image[style="background-image:url(https://placeimg.com/1000/500/arch)"]': 1
+    contains:
+      'bcl-banner__content': "EU Budget for the future"
+      'p': "Innovation, economy, environment and geopolitics"
+    equals:
+      'a': "Subscribe"
+image_shade_page_banner_centered_with_link_full_width_with_content_classes:
+  render:
+    '#type': pattern
+    '#id': banner
+    '#variant': 'image_shade'
+    '#fields':
+      settings:
+        'hero': false
+        'shade': true
+        'full_width': true
+        'centered': true
+      'title': "EU Budget for the future"
+      'description': "Innovation, economy, environment and geopolitics"
+      'link':
+        'label': "Subscribe"
+        'path': "https://example.com"
+      'image': "https://placeimg.com/1000/500/arch"
+      'content_classes': "content-class my-2"
+  assertions:
+    count:
+      'div[data-drupal-selector=banner]': 1
+      'div.bcl-banner.bg-lighter': 1
```

Comment:
Add classes: shade text-center full-width which are added because shade, full_width and centered are set as true. Check at netlify --> set the controls --> review the HTML generated.